mailman3_fedmsg_plugin.py

```diff
@@ -67,47 +78,44 @@ class Archiver(object):
         "user-agent",
     ]
 
-    def __init__(self):
-        if not getattr(getattr(fedmsg, '__local', None), '__context', None):
-            hostname = socket.gethostname().split('.')[0]
-            fedmsg.init(name="mailman.%s" % hostname)
-        self.config = fedmsg.config.load_config()
-
-
     def archive_message(self, mlist, msg):
```

Comment:
Method definition identical to https://github.com/fedora-infra/mailman3-fedmsg-plugin/blob/fedora-messaging/mailman3_fedmsg_plugin/__init__.py#L101